src/pymorize/units.py

```diff
@@ -79,9 +79,9 @@ def _get_units(
                 f" see dimensionless variable map for variable {cmor_variable!r}"
             )
     if from_unit is None:
-        raise ValueError("Unit not defined")
+        raise ValueError(f"Unit not defined: {from_unit=}")
     if not (to_unit or to_unit_alias):
-        raise ValueError("Unit not defined")
+        raise ValueError(f"Unit not defined: {to_unit=}, {to_unit_alias=}")
```

Comment:
To be considered: a better name of `to_unit_alias`, maybe something like `to_unit_from_dimless_table` or `to_unit_with_physically_complete_values`

Reply:
I went with `to_unit_dimensionless_mapping`. Are we happy with that?